embassy-nxp/src/pint.rs

```diff
@@ -294,49 +262,49 @@ fn handle_interrupt(interrupt_number: usize) {
 }
 
 #[allow(non_snake_case)]
-#[interrupt]
+#[cortex_m_rt::interrupt]
 fn PIN_INT0() {
     handle_interrupt(0);
 }
 
 #[allow(non_snake_case)]
-#[interrupt]
+#[cortex_m_rt::interrupt]
 fn PIN_INT1() {
     handle_interrupt(1);
 }
 
 #[allow(non_snake_case)]
-#[interrupt]
+#[cortex_m_rt::interrupt]
 fn PIN_INT2() {
     handle_interrupt(2);
 }
 
 #[allow(non_snake_case)]
-#[interrupt]
+#[cortex_m_rt::interrupt]
 fn PIN_INT3() {
     handle_interrupt(3);
 }
 
 #[allow(non_snake_case)]
-#[interrupt]
+#[cortex_m_rt::interrupt]
 fn PIN_INT4() {
     handle_interrupt(4);
 }
 
 #[allow(non_snake_case)]
-#[interrupt]
+#[cortex_m_rt::interrupt]
 fn PIN_INT5() {
     handle_interrupt(5);
 }
 
 #[allow(non_snake_case)]
-#[interrupt]
+#[cortex_m_rt::interrupt]
 fn PIN_INT6() {
     handle_interrupt(6);
 }
 
 #[allow(non_snake_case)]
-#[interrupt]
+#[cortex_m_rt::interrupt]
```

Comment:
You probably don't need to fully qualify the path these usages of `#[interrupt]`